### PR TITLE
Added a README.md file

### PR DESCRIPTION
**Pull request by citruscaviar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+```
+3. Run the application using Docker. Ensure that Docker is installed and running on your machine.
+Then, in the root directory of the project, run the following command:
+
+```bash
+docker-compose up
+```
+This will start the application along with the MySQL database in Docker containers.
+The application will be accessible at http://localhost:8080.
+4. Swagger:Once the application is running,
```
Comment (Elena-Bruyako) [inline README.md]: the 4 point should be at the new line
Screenshot 2024-11-15 at 13 04 08

File: README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- **JWT** (JSON Web Token for authentication)
+
+## How to Run the Project
+
+1. Clone this repository.
+```bash
+git clone https://github.com/citruscaviar/online-book-store
+```
+2. Set up the environment variables
+Create a **.env** file in the root directory of the project.
```
Comment (Elena-Bruyako) [inline README.md:59]: `Create a env file` should be at the new line
Screenshot 2024-11-15 at 13 04 10

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

**Review by ahoienko — APPROVED** [review]
Good job!

**Pull request merged**